src/Entity/CommerceContentEntityBase.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Drupal\commerce\Entity;
+
+use Drupal\Core\Entity\ContentEntityBase;
+use Drupal\Core\Language\LanguageInterface;
+
+/**
+ * Provides the base class for Commerce content entities.
+ */
+class CommerceContentEntityBase extends ContentEntityBase implements CommerceContentEntityInterface {
```

Comment:
Also, this only provides a public method for trnslated references. So really only relates to Product.